core/client/fs/src/test/java/alluxio/client/block/AlluxioBlockStoreTest.java

```diff
@@ -192,19 +187,18 @@ public void getOutStreamUsingLocationPolicy() throws Exception {
             .setLocationPolicy((workerOptions) -> {
               throw new RuntimeException("policy threw exception");
             });
-    mException.expect(Exception.class);
-    mBlockStore.getOutStream(BLOCK_ID, BLOCK_LENGTH, options);
+    assertThrows(Exception.class, () -> mBlockStore.getOutStream(BLOCK_ID, BLOCK_LENGTH, options));
   }
 
   @Test
   public void getOutStreamMissingLocationPolicy() throws IOException {
     OutStreamOptions options =
         OutStreamOptions.defaults(mClientContext).setBlockSizeBytes(BLOCK_LENGTH)
             .setWriteType(WriteType.MUST_CACHE).setLocationPolicy(null);
-    mException.expect(NullPointerException.class);
-    mException.expectMessage(
-        PreconditionMessage.BLOCK_WRITE_LOCATION_POLICY_UNSPECIFIED.toString());
-    mBlockStore.getOutStream(BLOCK_ID, BLOCK_LENGTH, options);
+    Exception e = assertThrows(NullPointerException.class,
+            () -> mBlockStore.getOutStream(BLOCK_ID, BLOCK_LENGTH, options));
+    assertTrue(e.getMessage()
+            .contains(PreconditionMessage.BLOCK_WRITE_LOCATION_POLICY_UNSPECIFIED.toString()));
```

Comment:
Since `ExpectedException.none()` is deprecated (highlighted in yellow in the IDE too), we should now change how we expected exceptions with a message. @apc999 Do you think this is acceptable? I'd like to first check with you before updating all the other tests.

Reply:
looks good to me